CMakeLists.txt

```diff
@@ -62,6 +60,17 @@ endif(CORECLR_SET_RPATH)
 OPTION(CLR_CMAKE_ENABLE_CODE_COVERAGE "Enable code coverage" OFF)
 OPTION(CLR_CMAKE_WARNINGS_ARE_ERRORS "Warnings are errors" ON)
 
+#----------------------------------------------------
+# Cross target Component build specific configuration
+#----------------------------------------------------
+if(CLR_CROSS_COMPONENTS_BUILD)
```

Comment:
Is this mostly consistency?

Reply:
Yes. I want to move to the runtime cmake defines instead of the old crusty ones.